src/System.Private.CoreLib/shared/System/Globalization/CultureData.Unix.cs

```diff
@@ -17,8 +18,8 @@ internal partial class CultureData
         const int ICU_ULOC_KEYWORD_AND_VALUES_CAPACITY = 100; // max size of keyword or value
         const int ICU_ULOC_FULLNAME_CAPACITY = 157;           // max size of locale name
         const string ICU_COLLATION_KEYWORD = "@collation=";
-        
-        
+
```

Comment:
nit: extra line.

Reply:
there were already 2, my editor just removed extra whitespace, will get rid of it though